src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const threeDaysOff = 20;
+  const sevenDaysOff = 50;
+
+  if (days < 3) {
+    return days * dayCost;
```

Comment:
You can also write this code - days * dayCost - into variable. It will be better to read. It just good code style, i think)

Reply:
yep, create a variable for `days * dayCost`, you repeat this three times